### PR TITLE
Saas 15.2 project update fix performance xbo

### PR DESCRIPTION
**Pull request by IT-Ideas**
This PR adresses performance issues on big databases in the project application, mainly focusing on the `project.project` form view as well as the `project.update` right side panel.

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/project/models/project.py [hunk 1]
```diff
@@ -1020,7 +1018,7 @@ def _read_group_personal_stage_type_ids(self, stages, domain, order):
     legend_blocked = fields.Char(related='stage_id.legend_blocked', string='Kanban Blocked Explanation', readonly=True, related_sudo=False)
     legend_done = fields.Char(related='stage_id.legend_done', string='Kanban Valid Explanation', readonly=True, related_sudo=False)
     legend_normal = fields.Char(related='stage_id.legend_normal', string='Kanban Ongoing Explanation', readonly=True, related_sudo=False)
-    is_closed = fields.Boolean(related="stage_id.fold", string="Closing Stage", related_sudo=False, help="Folded in Kanban stages are closing stages.")
+    is_closed = fields.Boolean(related="stage_id.fold", string="Closing Stage", store=True, index=True, related_sudo=False, help="Folded in Kanban stages are closing stages.")
```
Comment (KangOl) [inline addons/project/models/project.py:1021]: Does it really need to be stored? Doesn't use `auto_join=True` do the trick?

Comment (IT-Ideas) [reply addons/project/models/project.py:1021]: Hello @KangOl 👋🏼 
Are you still against it after having seen the explain plan I provided you with or can we go forward with this PR?

**Comment by IT-Ideas** [comment]
@robodoo rebase-ff

**Comment by IT-Ideas** [comment]
@robodoo r+

**Pull request closed without merging**